iroha_client/tests/account_with_multiple_signatories.rs

```diff
@@ -1,88 +1,85 @@
-#[cfg(test)]
-mod tests {
```

Comment:
Please live mod in all the tests as it was so the diff is better and we can actually understand what changed.